### DISCUSSION TITLE
Add support for pressing Enter to send message, Shift-Enter for next line

### DISCUSSION THREAD

**Discussion by ghost** [d99kris/nchat#254] [Ideas]
its very annoying to press ^x to send message. Can we just use Enter to send message and use Shift+Enter for next line

**Comment by d99kris** [comment]
Hi - In 6fd3675 / v4.90 I've added improved optional support for using enter key to send (and alt/opt-enter for linebreak when writing a message). Please see [FAQ for details](https://github.com/d99kris/nchat#2-send-messages-with-enter-key) for how to enable it.

**Reply by fulalas** [reply]
> Hi - In [6fd3675](https://github.com/d99kris/nchat/commit/6fd36756eef58d188dd1109803ddf401f5bc38fb) / v4.90 I've added improved optional support for using enter key to send (and alt/opt-enter for linebreak when writing a message). Please see [FAQ for details](https://github.com/d99kris/nchat#2-send-messages-with-enter-key) for how to enable it.

This is super great!

I have one question though: is it possible to paste a multi-line text without nchat breaking it in different messages?

**Reply by d99kris** [reply]
That's a good question. So if you use the internal paste function (alt/opt-v) it should work, as it inserts the clipboard text directly into the "compose buffer". However, if you use the OS/terminal paste (usually ctrl-shift-v on Linux or cmd-v on macOS) the text will be broken into different messages.
